Astral Pathfinder/UI/Map.cpp

```diff
@@ -29,7 +30,7 @@ Map::Map(Galaxy::Planet *planetArray) {
     
     int xpos = planets[0] -> getPosition().x;
     int ypos = planets[0] -> getPosition().y;
-    ship = new GameObject("Resources/Assets/simpleSpaceship.png",
+    ship = new Ship("Resources/Assets/simpleSpaceship.png",
```

Comment:
The map should contain a GameObject that represents the ship, the Ship class should not be a GameObject, unless we decide to include a gameObject instance in every other class